aws/ami/files/scylla_install_ami

```diff
@@ -60,64 +68,106 @@ if __name__ == '__main__':
         print('Error: need to specify --localrpm or --repo/--repo-for-install')
         sys.exit(1)
 
-    run('yum update -y')
+    if distro == 'centos':
+        run('yum update -y')
+
+        if args.repo_for_install:
+            run('curl -L -o /etc/yum.repos.d/scylla_install.repo {REPO_FOR_INSTALL}'.format(REPO_FOR_INSTALL=args.repo_for_install))
 
-    if args.repo_for_install:
-        run('curl -L -o /etc/yum.repos.d/scylla_install.repo {REPO_FOR_INSTALL}'.format(REPO_FOR_INSTALL=args.repo_for_install))
+        if args.localrpm:
+            rpms = glob.glob('{}/scylla*.*.rpm'.format(homedir))
+            run('yum install -y {}'.format(' '.join(rpms)))
+        else:
+            run('yum install -y {0}-machine-image {0}-debuginfo'.format(args.product))
 
-    if args.localrpm:
-        rpms = glob.glob('{}/scylla*.*.rpm'.format(homedir))
-        run('yum install -y {}'.format(' '.join(rpms)))
+        if args.repo_for_install:
+            os.remove('/etc/yum.repos.d/scylla_install.repo')
+        if args.repo_for_update:
+            run('curl -L -o /etc/yum.repos.d/scylla.repo {REPO_FOR_UPDATE}'.format(REPO_FOR_UPDATE=args.repo_for_update))
     else:
-        run('yum install -y {0}-machine-image {0}-debuginfo'.format(args.product))
+        run('apt-get update')
+        run('apt-get upgrade -y')
+        run('apt-get install gdebi-core -y')
+
+        if args.repo_for_install:
+            run('curl -L -o /etc/apt/sources.list.d/scylla_install.list {REPO_FOR_INSTALL}'.format(REPO_FOR_INSTALL=args.repo_for_install))
+            run('apt-get update --allow-insecure-repositories -y')
 
-    if args.repo_for_install:
-        os.remove('/etc/yum.repos.d/scylla_install.repo')
-    if args.repo_for_update:
-        run('curl -L -o /etc/yum.repos.d/scylla.repo {REPO_FOR_UPDATE}'.format(REPO_FOR_UPDATE=args.repo_for_update))
+        # gdebi can install only one package at once, and dependencies should
+        # resolved on every install, so we need to manually take care
+        # dependencies between scylla packages.
+        if args.localrpm:
+            pkgs = ['-conf', '-kernel-conf', '-python3', '-server', '-server-dbg', '-node-exporter', '-jmx', '-tools-core', '-tools', '-machine-image', '']
+            for pkg in pkgs:
+                deb = glob.glob('{}/{}{}_*.deb'.format(homedir, args.product, pkg))
+                run('gdebi -n {}'.format(deb[0]))
```

Comment:
- need to skip unexisting packages (for example dbg is not part of the ami and is not downloaded in the build ami)
- no need for empty string at the end (otherwise I don't understand the comment that we need to install packages one by one)
```suggestion
            pkgs = ['-conf', '-kernel-conf', '-python3', '-server', '-server-dbg', '-node-exporter', '-jmx', '-tools-core', '-tools', '-machine-image']
            for pkg in pkgs:
                deb = glob.glob('{}/{}{}*.deb'.format(homedir, args.product, pkg))
                if deb:
                    run('gdebi -n {}'.format(deb[0]))
                else:
                    print(f"{args.product}{pkg} deb package file not found, skipping.")
```

Reply:
> * need to skip unexisting packages (for example dbg is not part of the ami and is not downloaded in the build ami)

Well, we decided to install debuginfo package since Scylla-3.2, so I suppose we should do same in Ubuntu AMI:
https://github.com/scylladb/scylla/commit/30414d9c23a8efa233fec112467095261faefcef

> * no need for empty string at the end (otherwise I don't understand the comment that we need to install packages one by one)

I know this line feels bit strange, what we want to do here is:
```
gbebi -n ${product}-conf_*.deb
gbebi -n ${product}-kernel-conf_*.deb
....
gdebi -n ${product}-machine-image_*.deb
gdebi -n ${product}_*.deb
```
So the last empty item is "${product}".
Maybe we should do something like this instead:
```
pkgs = ['{}-conf'.format(product), '{}-kernel-conf'.format(product), ...  '{}-machine-image'.format(product), product]
```
It's verbose, but probably more understandable.

Reply:
Stopped using gdebi, switched to local apt repository. So dropped these lines.